<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Sudhakar" failures="1" tests="3" name="Test" time="16.262" errors="0" timestamp="2024-10-23T08:36:42 IST">
  <testcase classname="Day46.OrangeHRM" name="testlogo" time="0.079"/>
  <testcase classname="Day46.OrangeHRM" name="testurl" time="0.015">
    <failure type="java.lang.AssertionError" message="expected [https://opensource-demo.orangehrmlive.com/] but found [https://opensource-demo.orangehrmlive.com/web/index.php/auth/login]">
      <![CDATA[java.lang.AssertionError: expected [https://opensource-demo.orangehrmlive.com/] but found [https://opensource-demo.orangehrmlive.com/web/index.php/auth/login]
at Day46.OrangeHRM.testurl(OrangeHRM.java:39)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- testurl -->
  <testcase classname="Day46.OrangeHRM" name="testpagetitle" time="0.001">
    <skipped/>
  </testcase> <!-- testpagetitle -->
</testsuite> <!-- Test -->
